src/Game.java

```diff
@@ -0,0 +1,149 @@
+            while (option != EXIT) {
+
+                System.out.println("Wybierz opcję:");
+                System.out.println("1 - graj");
+                System.out.println("2 - koniec programu");
+                option = scanner.nextInt();
+                scanner.nextLine();
+
+                switch (option) {
+                    case EXIT:
+                        System.out.println("Bye bye!");
+                        scanner.close();
+                        break;
+                    case PLAY:
+                        play();
+                        break;
+                    default:
+                        System.out.println("Wybrałeś nieprawidłową opcję");
+                }
+            }
+            isException = false;
+        } catch (InputMismatchException e) {
+            System.out.println("Podałeś niepoprawny znak.");
+            scanner.nextLine();
+        }
+    }
+}
+
+    private String nextWordToFind() {
+        return Keywords.WORDS[RANDOM.nextInt(Keywords.WORDS.length)];
+    }
+
+    private void addWord(String word) {
+        lettersFromFile.add(word);
+    }
+
+    private String wordToFindFromFile() {
+        String fileName = "words.txt";
+        try (
+                var fileReader = new FileReader(fileName);
+                var bufferedReader = new BufferedReader(fileReader);
+        ) {
+            while (bufferedReader.readLine() != null) {
+                addWord(bufferedReader.readLine());
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+        return lettersFromFile.get(RANDOM.nextInt(lettersFromFile.size())).toUpperCase();
+    }
+
+    private void newGame() {
+        nbErrors = 0;
+        letters.clear();
+        wordToFind = nextWordToFind();
+        wordFound = new char[wordToFind.length()];
+
+        Arrays.fill(wordFound, '*');
+    }
+
+    private boolean wordFound() {
+        return wordToFind.contentEquals(new String(wordFound));
+    }
+
+    private String wordFoundContent() {
+        StringBuilder sb = new StringBuilder();
+
+        for (int i = 0; i < wordFound.length; i++) {
+            sb.append(wordFound[i]);
+
+            if (i < wordFound.length - 1) {
+                sb.append(" ");
+            }
+        }
+        return sb.toString();
+    }
+
+    private void enter(String c){
+        if(!letters.contains(c)){
+            if (wordToFind.contains(c)){
+                int index = wordToFind.indexOf(c);
+
+                while (index >= 0){
+                    wordFound[index] = c.charAt(0);
+                    index = wordToFind.indexOf(c,index + 1);
+                }
+            } else {
+                nbErrors++;
+            }
+            letters.add(c);
+        }
+    }
+
+    private void play(){
+        newGame();
+        try {
+            while (nbErrors < MAX_ERRORS) {
+                System.out.println("Hasło do zgadnięcia: " + '\n' + wordFoundContent());
```

Comment:
Haslo do zgadniecia? What does it mean? Are you printing the word that user needs to guess?

Reply:
Prints '*' instead of letters in the word. This is in accordance with the instructions contained in the issue

Reply:
got it